h/static/scripts/group-forms/components/AnnotationCard.tsx

```diff
@@ -129,6 +132,13 @@ export default function AnnotationCard({
 
   const annoQuote = quote(annotation);
 
+  // Should the annotation body be rendered in a collapsed state, assuming it is
+  // collapsible (exceeds allotted collapsed space)?
+  const [annoBodyCollapsed, setAnnoBodyCollapsed] = useState(true);
```

Comment:
Given the context I think the "anno" prefix on these variable names is redundant. `bodyCollapsed` would be fine.